api/MediaQueryList.json

```
@@ -97,7 +99,7 @@
       },
       "EventTarget_inheritance": {
         "__compat": {
-          "description": "<code>MediaQueryList</code> inheriting from <code>EventTarget</code>",
+          "description": "<code>MediaQueryList</code> is based on <code>EventTarget</code>",
```

Can you elaborate on your thinking by the change here a little? I see that the original is a little awkward, but reading the spec it seems like inheritance is the right term. Perhaps something like this is good?

```suggestion
          "description": "<code>MediaQueryList</code> inherits <code>EventTarget</code>",
```

Or maybe I'm missing something and the relationship is weirder than I thought.

---

I don’t recall making that change, though clearly  I did. I’ll undo it.